.travis/before_install.sh

```diff
@@ -119,7 +119,8 @@ case "${BUILD_TARGET}" in
         installdeps "happy-deps"
 
         cd $HOME
-        git clone https://github.com/openweave/happy.git
+        git clone https://github.com/jenniexie/happy.git
+        #git clone https://github.com/openweave/happy.git
```

Comment:
I have just changed back to git clone happy master for this one because I merged my changes to happy master:
 " git clone https://github.com/openweave/happy.git "